MANIFEST.in

```diff
@@ -14,6 +14,7 @@ recursive-include docs *
 recursive-include res *
 recursive-include scripts *
 recursive-include scripts-dev *
+recursive-include synapse *.pyi
```

Comment:
It seems to list the files which go into the `sdist`?

I would certainly think that the `.pyi` should be part of the source distribution.